### PR TITLE
Add FA, Add TW, enable composition API

### PR DESCRIPTION
**Pull request by unrevised6419**

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 module.exports = {
   "stories": [
     "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+    "../src/**/*.stories.@(js|jsx|ts|tsx)",
+    "../src/**/**/*.stories.@(js|jsx|ts|tsx)"
```
Comment (unrevised6419) [inline .storybook/main.js]: @YoSoySergio is this needed?

Comment (YoSoySergio) [reply .storybook/main.js]: yes, if we gonna keep the stories inside component folder, no?

**Review by milosodalovic — APPROVED** [review]
Looks good 👍

File: .storybook/main.js (hunk 1 above)
Comment (unrevised6419) [reply .storybook/main.js]: `"../src/**/*.stories.@(js|jsx|ts|tsx)"` is enough, `**` means any nested folders

Comment (YoSoySergio) [reply .storybook/main.js]: I thought its just 1st level ones, if it'll work without it - feel free to remove that line

**Pull request merged**